docs/PreReleaseNotes.txt

```diff
@@ -31,6 +31,8 @@ Prerelease Notes
   **Commit: 0f79a38
   **[XrdMacaroons] Fix authentication when different tokens are used in the same TCP session**
   **Commit: 4410d56
+  **[XrdHttp] 500 Internal Server Error on unknown checksum algorithm**
```

Comment:
That really is not a meaningful summary of the changes. How about "Follow RFC checksum requirements in XRootD http".